src/main/java/core/basesyntax/Draw.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Draw {
```

Comment:
Consider renaming the interface to `Drawable` to better fit the convention of using adjectives for interface names. 😊